protobuf/xain/grpc/coordinator.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package xain.protobuf.coordinator;
+
+service Coordinator {
+  rpc Rendezvous(google.protobuf.Empty) returns (RendezvousReply) {}
```

Comment:
I'd propose to introduce a `RendezvousRequest` as a container for e.g. information the client would like to share.

Reply:
:+1: but I will leave the changes to `setup.py` so that we have access to google defined types if needed. I think it's helpful